lstchain/visualization/plot_drs4.py

```diff
@@ -79,8 +80,8 @@ def plot_pedestals(data_file, pedestal_file, run=0 , plot_file="none", tel_id=1,
             ped_data = event.mon.tel[tel_id].pedestal
             break
 
-    camera = CameraGeometry.from_name("LSTCam", 2)
-
+    camera = reader.subarray.tels[tel_id].camera.geometry
```

Comment:
`camera_geometry` would be more explicit,  especially now that they are different objects in ctapipe

Reply:
solved